Constitution.txt

```diff
@@ -1,490 +1,264 @@
-That after such Publication the Electors should be appointed, and the Senators and Representatives elected: That the Electors should meet on the Day fixed for the Election of the President, and should transmit their Votes certified, signed, sealed and directed, as the Constitution requires, to the Secretary of the United States in Congress assembled, that the Senators and Representatives should convene at the Time and Place assigned; that the Senators should appoint a President of the Senate, for the sole Purpose of receiving, opening and counting the Votes for President; and, that after he shall be chosen, the Congress, together with the President, should, without Delay, proceed to execute this Constitution.
+2: If the exiting State has land borders with a territory which is not a member of the Union, or has water access to the Oceans of the World through waters not claimed as territory by any other State of the Union, the State shall be considered to be sovereign and foreign from these United States on the date one year after the President of these United States has been informed of the State's intent to leave the Union.
 
- 	
-By the unanimous Order of the Convention
+3: Otherwise, the exiting state shall be considered sovereign and foreign on the date two years after the President of these United States has been informed.
 
-W. Jackson  Secretary.
+4: During the period of time between the election of the State to leave the Union and the date of exit, the government of the State shall engage in negotiation of treaties to be activated on the date of exit, to govern trade between the newly separated State and the Union, and to come to agreement regarding the ownership of such lands within the borders of the newly separated State which previously had been purchased for use by the Government of these United States. For the purpose and within the scope of these negotiations, the State shall immediately be considered sovereign and foreign from these United States.
 
-Go: Washington -Presidt.
+5: A State which so elects to leave the Union shall be treated as a friend and neighbor unless the State's actions shall decry such treatment; but the State shall be forbidden from rejoining the Union for a period of twenty-four years.
 
-Letter of Transmittal to the President of Congress
-skip to Amendments   up to Letter of Transmittal
-In Convention.  Monday September 17th 1787.
-SIR: 
+6: The Union may begin the process of removing a State or several States from the Union. In this case, the period until exit shall begin when the President of these United States delivers a document, signed by all Representatives and Senators, and all Chief Executives of the States, except those of the State or States being removed, to the Chief Executives of each State being removed. Afterwards, the negotiations shall proceed as if the State or States being removed had elected to being the process of leaving the Union.
```

Comment:
If one State can secede, so too should the Union be able to eject a State or group of States, if their inclusion is to the detriment of the Union. If this happens, the structure of the Union will continue, and those ejected States will still be peaceful friends, unless the ejected States are so offended as to declare war.